react/src/components/CreateMapModal/CreateMapModal.tsx

```diff
@@ -80,8 +85,8 @@ const CreateMapModal = ({ isOpen, toggle }: CreateMapModalProps) => {
     handleCreateProject(projectData);
   };
   return (
-    <Modal isOpen={isOpen} toggle={toggle}>
-      <ModalHeader toggle={toggle}>Create a New Map</ModalHeader>
+    <Modal isOpen={isOpen} toggle={handleClose}>
```

Comment:
@tjgrafft, regarding:
> custom css rules to handle smaller screen sizes

If this minor issue is explored, and successful CSS changes are not forthcoming, consider a wider modal.

| XL Modal | Size Option |
| - | - |
| <img width="1194" alt="xl modal" src="https://github.com/TACC-Cloud/hazmapper/assets/62723358/a8a10e4a-da25-41c3-9438-d2951a77230e"> | <img width="1194" alt="size" src="https://github.com/TACC-Cloud/hazmapper/assets/62723358/f3053d1b-dd50-4acf-bdd5-7d5e1bd9b342"> |

- https://reactstrap.github.io/?path=/docs/components-modal--modal
- https://react-bootstrap.netlify.app/docs/components/modal/#optional-sizes

_P.S. I think all the elements in the modal content should be able to flexible enough to not need media queries._

Reply:
I took a stab at the .hazmapper suffix while fixing the horizontal scroll and handling the different screen sizes--see screenshot below. Probably not a finished product and needs some polishing, but it's a start. If that code and the UI look acceptable, do you mind being the second approver for this PR? Once it's approved I'll merge it and handle any future tweaks in the March sprint. Thanks!
![Screenshot 2024-02-28 at 1 40 47 PM](https://github.com/TACC-Cloud/hazmapper/assets/50084480/9333e596-02c1-40a8-a4c3-2c9d967e68bf)

Reply:
Oh, hey, cool. No problem. I only just confirmed you already handled everything I asked about. Sorry for the delay. Thanks for all the effort.

Reply:
All good, thanks for taking the time to review this PR and helping me with the UI for this modal/form. I appreciate it!